### ISSUE TITLE
Add a .desktop file

### ISSUE THREAD

**Issue by cippaciong** [SuperDisk/hUGETracker#169]
Hello, it would be nice if the Linux package shipped a `hugetracker.desktop` file, so that the application would show up also in desktop environment application menus or application launchers like Rofi.

I'm testing it locally with a manually created desktop file and it's working fine. I'm just not sure what's the correct value for the `Icon` field and if we should add anything else to `Categories` to support multiple desktop environments.

Here's the file:
```ini
[Desktop Entry]
Name=hUGETracker
Comment=The music composition suite for the Nintendo Game Boy
GenericName=hUGETracker
Exec=hugetracker %U
Icon=hugetracker
Type=Application
StartupNotify=true
Categories=GNOME;GTK;Utility;
```

**Comment by ISSOtm** [comment]
Good idea! I just think that this file should be tweaked further. It's also a good idea to install some MIME type detection files to integrate even better; you can look into [how SameBoy does it](https://github.com/LIJI32/SameBoy/blob/64cf389edfc331e615a9571c1682f96551a5b5b8/Makefile#L705-L726) (and all involved files) for inspiration.

**Comment by SuperDisk** [comment]
Since I just distribute a .zip file with the Linux binary, I can't really package a .desktop file that would actually work since the executable isn't at a fixed location. I think providing a .desktop is in the hands of distro packagers but correct me if I'm wrong. I added your .desktop in 12003cdab1dc6ba96ae04b4b5edb80aa9942f9f8 for packagers to use, though AFAIK nobody packages HT for any distros aside from ISSO doing it for Arch.
